www/js/app.js

```diff
@@ -768,10 +833,16 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
                 $('#articleListHeaderMessage').empty();
                 $('#articleListWithHeader').hide();
                 $("#prefix").val("");
-                iframeArticleContent.onload = function () {
+                iframeArticleContent.onload = function() {
                     // The content is fully loaded by the browser : we can hide the spinner
-                    iframeArticleContent.onload = function () {};
                     $("#searchingArticles").hide();
+                    // Deflect drag-and-drop of ZIM file on the iframe to Config
+                    var docBody = iframeArticleContent.contentDocument.documentElement.getElementsByTagName('body')[0];
+                    docBody.addEventListener('dragover', handleIframeDragover);
```

Comment:
Maybe it would be safer to check that all intermediate variables are defined before using them.

Reply:
OK, will do. You're right, we could get unhandled exceptions.